torchreid/engine/engine.py

```diff
@@ -334,22 +334,7 @@ def run(
                                      test_only = True
                                     )
             return test_results
-
-        if not lr_finder:
-            print('Test before training')
```

Comment:
Maybe we do the same with the test_only case? And move it to the run_training function? This will help us to remove test_only parameters and few lines of code inside run finction